SimDataFormats/Associations/plugins/BuildFile.xml

```diff
@@ -1,5 +1,8 @@
 <use name="FWCore/Framework"/>
 <use name="FWCore/ParameterSet"/>
+<use name="DataFormats/HGCalReco"/>
+<use name="DataFormats/ForwardDetId"/>
+<use name="RecoLocalCalo/HGCalRecAlgos"/>
```

Comment:
does https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCreatingNewProducts#Restrictions_on_the_package_defi
still apply to  SimDataFormats/Associations?

One case/example is a plugins/ is not expected

Reply:
its true that SimDataFormats/Associations having a plugins directory doesn't make much sense. That is already the case, however

Reply:
but "already the case" seems to be a recent addition #32427 (not something from long time ago)

Reply:
can this plugin be decoupled from the DataFormats now or is there a strong preference to do it in a follow up PR?